protocols/raft/src/main/java/io/atomix/protocols/raft/proxy/impl/RaftProxyInvoker.java

```diff
@@ -128,7 +128,7 @@ private void invokeQuery(RaftOperation operation, CompletableFuture<byte[]> futu
         .withSession(state.getSessionId().id())
         .withSequence(state.getCommandRequest())
         .withOperation(operation)
-        .withIndex(Math.max(state.getResponseIndex(), state.getEventIndex()))
+        .withIndex(state.getResponseIndex())
```

Comment:
I'm not entirely sure how this got in here :-(

Reply:
https://github.com/atomix/atomix/pull/419